packages/compiler/test/server/completion.test.ts

```diff
@@ -206,7 +206,7 @@ describe("compiler: server: completion", () => {
         documentation: {
           kind: MarkupKind.Markdown,
           value:
-            "```cadl\ndec doc(target: unknown, doc: Cadl.string, formatArgs?: Cadl.object)\n```",
+            "```cadl\ndec Cadl.doc(target: unknown, doc: Cadl.string, formatArgs?: Cadl.object)\n```",
```

Comment:
```suggestion
            "```cadl\ndec Cadl.doc(target: unknown, doc: Cadl.string, formatArgs?: Cadl.object)\n```",
```

Wondering if we omit the Cadl. namespace from all the built-ins when rendering the type. Same with getTypeName

Reply:
We should, I started doing this and broke some things and decided that since we already had some Cadl. better to stop here and do it separately.

Reply:
#1348